### PR TITLE
eslint: error when jsx key is missing

### PR DESCRIPTION
**Pull request by akgagnat**

### CODE REVIEW

**Comment by akgagnat** [comment]
[Eslint feiler når react key mangler ved iterations](https://trello.com/c/9725R8XI/1387-eslint-feiler-n%C3%A5r-react-key-mangler-ved-iterations)

File: .eslintrc.json [hunk 1]
```diff
@@ -5,6 +5,7 @@
     "no-unused-vars": "off",
     "no-duplicate-imports": "error",
     "@typescript-eslint/no-unused-vars": ["error", { "args": "none" }],
+    "react/jsx-key": [2, { "checkFragmentShorthand": true }],
```
Comment (lotorvik) [inline .eslintrc.json:8]: Kan du bruke error?

**Review by lotorvik — APPROVED** [review]
👍

File: .eslintrc.json (hunk 1 above)
Comment (akgagnat) [reply .eslintrc.json]: ja, endret til error

**Pull request closed without merging**